### PR TITLE
Add cypress tests for OpFab started in another tab (#3103)

### PR DESCRIPTION
**Pull request by olivierPigeon-RTE**
Signed-off-by: olivierPigeon-RTE <olivier.pigeon@rte-france.com>

Nothing in release notes

### CODE REVIEW

File: src/test/cypress/cypress/integration/CheckIfOpfabUrlIsAlreadyTaken.spec.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ * SPDX-License-Identifier: MPL-2.0
+ * This file is part of the OperatorFabric project.
+ */
+
+/* This test file focuses on some state-type specific behaviour in card details header. As the Cypress test suite grows,
+it might make sense to merge it with other tests.
+* */
```
Comment (freddidierRTE) [inline src/test/cypress/cypress/integration/CheckIfOpfabUrlIsAlreadyTaken.spec.js:12]: Remove comment not related to these tests

File: src/test/cypress/cypress/integration/CheckIfOpfabUrlIsAlreadyTaken.spec.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+describe('Card detail', function () {
+
```
Comment (freddidierRTE) [inline src/test/cypress/cypress/integration/CheckIfOpfabUrlIsAlreadyTaken.spec.js:14]: Change describe text to reflect the current tests

File: src/test/cypress/cypress/integration/CheckIfOpfabUrlIsAlreadyTaken.spec.js [hunk 3]
```diff
@@ -0,0 +1,67 @@
+/* Copyright (c) 2021-2022, RTE (http://www.rte-france.com)
```
Comment (freddidierRTE) [inline src/test/cypress/cypress/integration/CheckIfOpfabUrlIsAlreadyTaken.spec.js:1]: Copyright is only 2022

**Review by freddidierRTE — CHANGES_REQUESTED** [review]

File: src/test/cypress/cypress/integration/CheckIfOpfabUrlIsAlreadyTaken.spec.js (hunk 3 above)
Comment (olivierPigeon-RTE) [reply src/test/cypress/cypress/integration/CheckIfOpfabUrlIsAlreadyTaken.spec.js]: done

**Pull request merged**